### DISCUSSION TITLE
Is running LogicApps as `dotnet-isolated` a supported feature?

### DISCUSSION THREAD

**Discussion by earloc** [Azure/logicapps#816] [Help Needed]
Hi.

We´re currently developing some line-of-business (LOB) custom-connectors in order to be able to automate various tasks in our business.
One thing we didn´t find anywhere documented / sampled is a way to "hook" into a workflows "lifecycle"-events, such as `started`, `ended` (or `failed`, maybe). 

As there may be other things that´d be good to have finer grained control over, I started to try to host LogicApps (Standard) in a `dotnet-isolated`-mode, following [these instructions here](https://learn.microsoft.com/th-th/azure/azure-functions/dotnet-isolated-process-guide).
Current hope is that this way we could levereage some `Middleware` to achieve the initial goal, as well as having complete control over the apps start-process, etc. seems like a nice benefit, also.

Sadly, I´m facing various issues when trying to get this running. I´ve got the feeling that I´d "just" need to reflect the same startup-configuration which `azure functions core tools` does in order to get LogicApps (Standard) in `dotnet`-mode up and running, but currently am missing the needed pieces / directions.

So:
1. Is `dotnet-isolated` a supported mode to host LogicApps (Standard) ? If so, are there any directions on how to configure the necessary services, etc. in `program.cs`?
2. Is there maybe another way to "hook" into described "lifecycle"-events?

**Comment by earloc** [comment]
transferred to this [Q&A Question](https://learn.microsoft.com/en-us/answers/questions/1316152/is-running-logicapps-as-dotnet-isolated-a-supporte)

**Reply by earloc** [reply]
What does one need to do to get reactions to discussions, here @divyaswarnkar ?

**Reply by earloc** [reply]
A bummer that no one here cares to leave some kind of reply 🙁.

**Comment by earloc** [comment]
Over at the [Q&A Question](https://learn.microsoft.com/en-us/answers/questions/1316152/is-running-logicapps-as-dotnet-isolated-a-supporte) there has been some activity, recently, which led me to the conclusion that the answer to my original question should be a NO, at least from the perspective of having full control over the apps startup (and to do things like adding `middlewares`, etc.).

TLDR; 
Especially regarding `actions` executed within a `stateful` workflow which might have scaled out, it is really not guaranteed on which "physical" process such an action is executed. Trying to spin up a "scope" around a workflow´s execution might therefore not scale.
